docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,266 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fast Fourier Transform (FFT)\n",
+    "The following gives background information that is helpful to understand how the Fast Fourier Transform (FFT) and the corresponding normalizations are defined in pyfar and how these are related to the concepts of energy and power signals."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## FFT definition\n",
+    "The discrete Fourier spectrum of an arbitrary, but band-limited signal $x(n)$ is defined as\n",
+    "$$ X(k) = \\sum_{n=0}^{N-1} x(n) e^{-i 2 \\pi k \\frac{n}{N}}$$\n",
+    "using a negative sign convention in the transform kernel $e^{-i 2 \\pi k \\frac{n}{N}}$, and normalized angular frequency $\\omega_k = 2 \\pi k / N$. Analogously, the discrete inverse Fourier transform is implemented as\n",
+    "\n",
+    "$$ x(n) = \\frac{1}{N} \\sum_{k=0}^{N-1} X(k) e^{i2\\pi k \\frac{n}{N}}$$\n",
+    "\n",
+    "Real-valued time signals result in Fourier spectra with complex conjugate symmetry for negative and positive frequencies $X(k) = X(-k)^*$. In this case, the left-hand side of the spectrum is discarded, and only the right-hand side $X_R(k) = X(k)~\\forall~0 \\le k \\le \\lfloor N/2 \\rfloor$ is saved."
```

Comment:
here we could add a short example to demonstrate it, sth like:

```
import pyfar as pf
import numpy as np

sig = pf.signals.sine(1000, 44100)
pf.plot.time_freq(sig)
```
we could also change the explaination and refer to what we see.